pkg/resources/validation/roletemplate/roletemplate_test.go

```diff
@@ -0,0 +1,142 @@
+package roletemplate
+
+import (
+	"strconv"
+	"testing"
+
+	v3 "github.com/rancher/rancher/pkg/apis/management.cattle.io/v3"
+	controllerv3 "github.com/rancher/webhook/pkg/generated/controllers/management.cattle.io/v3"
+	"github.com/rancher/webhook/pkg/mocks"
+	rbacv1 "k8s.io/api/rbac/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var defaultRules = []rbacv1.PolicyRule{
+	{
+		APIGroups: []string{""},
+		Resources: []string{"pods"},
+		Verbs:     []string{"get", "list", "watch"},
+	},
+}
+
+var circleRoleTemplateName = "circleRef"
+
+func Test_checkCircularRef(t *testing.T) {
```

Comment:
The name, by convention, would just be `TestCheckCircularRef` in this case.

Reply:
That's not how at least one of the other webhook's tests are named. See: https://github.com/rancher/webhook/blob/53b1ff449ed461449beeca22b57e7aa212052c8d/pkg/resources/validation/cluster/provisioningcluster_test.go#L7 .

I can switch it to the standard go testing convention, but I think this discrepancy should be noted.

Reply:
Yeah, it's all immaterial really, that other example is OK despite not following the convention, but I don't believe that convention is strict...